src/CMakeLists.txt

```diff
@@ -231,7 +231,25 @@ install(FILES
     ${CMAKE_BINARY_DIR}/exiv2lib_export.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/exiv2)
 
-install(EXPORT exiv2Config DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2")
+# CMake configs
+set(export_dir "${CMAKE_BINARY_DIR}/export")
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+    "${export_dir}/exiv2ConfigVersion.cmake" COMPATIBILITY AnyNewerVersion
+)
+
+configure_package_config_file(
+    "${CMAKE_SOURCE_DIR}/cmake/exiv2Config.cmake.in"
+    "${export_dir}/exiv2Config.cmake"
+    INSTALL_DESTINATION "share/exiv2/cmake"
```

Comment:
Please, keep the usage of the variable `${CMAKE_INSTALL_LIBDIR}` instead of using directly **lib** or **share**. It helps to have better support for different linux distributions.

Reply:
Oh cool. I wasn't aware of this variable. Done.

Reply:
I did similar changes in 0.27. What if merge them before this one?

Reply:
Do you mean #672? We can forward port it. Does something differently than the commit in this PR?

Reply:
@D4N yes, I meant that one. Differences are minor. I've created another PR for master.